cloud-interfaces/api/cloudfiles-api.rst

```diff
@@ -115,10 +116,10 @@ In the sample response, ``bytes`` and ``name`` correspond
 to the information available in the Cloud Control Panel.
 
 In the :rax-docs:`Getting Started Guide for the Cloud Files API
-<files/api/v1/cf-getting-started/content/Overview-d1e01.html>`,
-you can see an example of
-:rax-docs:`obtaining a list of account details and Cloud Files containers by using the cURL command-line interface (CLI)
-<files/api/v1/cf-getting-started/content/Show-Account-Details-d1e101.html>`.
+<cloud-files/v1/getting-started/use-API-directly/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Creating a storage container
+<cloud-files/v1/getting-started/use-API-directly/#creating-a-storage-container>`.
 
 Learn about Cloud Files in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There are 2 links to SDK QuickStart for Cloud Files in this section that give a page not found message.